docs/guides/contributing.rst

```diff
@@ -72,6 +72,16 @@ Python "venv" with all dependencies and commands installed into it.
    Python virtual environment.
 
    It will also install libraries used during development.
+   
+#. Build documentation
```

Comment:
I'd make this into a top-level section after "dev server"

Reply:
Fair. Addressed as a new top-level section with a little more clarification on what EdgeDB uses for documentation (file syntax)